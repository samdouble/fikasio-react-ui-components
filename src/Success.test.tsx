import React from 'react';
import { render } from '@testing-library/react';
import { Success } from '.';

describe('Success', () => {
  it('Renders correctly', () => {
    const { baseElement } = render(
      <Success>
        It worked
      </Success>,
    );
    expect(baseElement).toMatchSnapshot();
  });
});
