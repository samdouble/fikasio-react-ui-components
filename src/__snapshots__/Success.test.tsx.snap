// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success Renders correctly 1`] = `
<div
  className="fikasio-success fikasio-theme-dark"
  style={Object {}}
>
  <div
    className="fade alert alert-success show"
    role="alert"
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-check fa-lg "
      data-icon="check"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={
        Object {
          "fontSize": 16,
          "marginRight": 10,
        }
      }
      viewBox="0 0 448 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
